libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -107,6 +107,38 @@ string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata)
 	});
 }
 
+string YulUtilFunctions::copyToMemoryLiteralFunction(string const& _literal)
+{
+	solAssert(!_literal.empty(), "");
+	string functionName = "copy_literal_to_memory_" + _literal;
```

Comment:
`_literal` has to be escaped somehow - or use the hash.

Reply:
I will use hash, fixing now